source/extensions/filters/http/transformation/transformation_filter_config.cc

```diff
@@ -70,10 +71,21 @@ TransformationFilterConfig::TransformationFilterConfig(
               fmt::format("Failed to parse response template: {}", e.what()));
         }
       }
+      if (route_transformation.has_on_stream_completion_transformation()) {
+        try {
+          on_stream_completion_transformation = Transformation::getTransformer(
+              route_transformation.on_stream_completion_transformation(), context);
+        } catch (const std::exception &e) {
+          throw EnvoyException(
+              fmt::format("Failed to parse on stream completion template: {}", e.what()));
```

Comment:
This error message isn't necessarily accurate. This only applies to the inja transformer, we should make this more generic